src/charm.py

```diff
@@ -584,11 +581,22 @@ def _start_server(self):
 
             if not self._wp_is_installed():
                 self._wp_install()
-            if self._current_wp_config() is None:
-                # For security reasons, never start WordPress server if wp-config.php not exists
-                raise FileNotFoundError(
-                    "required file (wp-config.php) for starting WordPress server not exists"
+        else:
+            for _ in range(60):
```

Comment:
DO you think it would be worth updating the status message of the charm to indicate it is waiting for the database to be ready?

Reply:
Added, thanks!